changes.go

```diff
@@ -51,9 +51,11 @@ func (c *pollChangeManager) close() <-chan bool {
 	}
 	// not launched
 	if atomic.LoadInt32(c.isLaunched) == int32(0) {
-		log.Error("pollChangeManager: close() called when pollChangeWithBackoff unlaunched! close tokenManager!")
+		log.Error("pollChangeManager: close() called when pollChangeWithBackoff unlaunched! Will wait until pollChangeWithBackoff is launched and then kill it and tokenManager!")
```

Comment:
This message is being surfaced as an error, so it's user facing.  Probably best not to expose method internals to users.  Maybe this statement should be a Debug, and then you could add and Error log saying something like "Attempt to close unstarted change manager"